dynamo/dynamo_logger.py

```diff
@@ -189,18 +189,18 @@ def report_progress(self, percent=None, count=None, total=None, progress_name=""
 
     def finish_progress(self, progress_name="", time_unit="s", indent_level=1):
         self.log_time()
-        self.report_progress(percent=100, progress_name=progress_name)
+        #self.report_progress(percent=100, progress_name=progress_name)
 
         saved_terminator = self.logger_stream_handler.terminator
         self.logger_stream_handler.terminator = ""
-        self.logger.info("\n")
+        #self.logger.info("\n")
```

Comment:
In lines 192 and 196, maybe `# ` is better than `#` according to the code style guide.